java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -0,0 +1,63 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.BitcountOptions;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Supports commands and transactions for the "Bitmap Commands" group for standalone and cluster
+ * clients.
+ *
+ * @see <a href="https://redis.io/docs/latest/commands/?group=bitmap">Bitmap Commands</a>
+ */
+public interface BitmapBaseCommands {
+    /**
+     * Counts the number of set bits (population counting) in a string. Returns zero if the key is
```

Comment:
Move second sentence to `@return` section or drop

Reply:
Thanks! Addressed in https://github.com/Bit-Quill/glide-for-redis/pull/263/commits/2575b0125be5640b722ee685c33c9fd624ae1b00